### ISSUE TITLE
Supported or optional?

### ISSUE THREAD

**Issue by alice-mkh** [danrabbit/linuxisnotaboutsystray#3]
![Screenshot from 2019-09-10 21-25-15](https://user-images.githubusercontent.com/1510457/64631921-a6272480-d3e7-11e9-9c6e-a7bf9b6e89f9.png)

**Comment by alice-mkh** [comment]
And in that case I wonder why MPRIS covers are optional in GNOME :)

**Comment by cassidyjames** [comment]
Yeah we need to clarify this. The idea was that a whole row could be considered optional, so supported features would be blue instead of green, and unsupported ones would be gray instead of red. But it's not super clear.

**Comment by alice-mkh** [comment]
Oh. That's not clear indeed.
